_docs-sources/guides/welcome.md

```diff
@@ -3,6 +3,24 @@
 "sidebar_position": 1
 ---
 
-# Welcome
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
 
-This is an introduction to the guides section
\ No newline at end of file
+# Gruntwork Guides
+
+As a Gruntwork subscriber, you have two primary ways to engage with our library. Whichever approach you choose, our guides provide step-by-step instructions to help you deploy your infrastructure with confidence. We also offer detailed upgrade guides to ensure that your infrastructure remains up-to-date with the latest security patches, compliance requirements, and DevOps best practices.
```

Comment:
I think we should be explicit here about what those two ways are: 
- ref arch
- built it yourself

Reply:
I guess the buttons are there - but my reptilian brain is still getting hung up on the fact that I didn't get info about what the two ways are right away.

Reply:
We captured this when discussing it live